src/APICalls.js

```diff
@@ -0,0 +1,11 @@
+const baseURL = 'https://rancid-tomatillos.herokuapp.com/api/v2/movies'
+
+export const fetchAllMovies = () => {
+  return fetch(`${baseURL}`)
+    .then(response => response.json())
+}
+
+export const fetchMovieDetails = (id) => {
+  return fetch(`${baseURL}/${id}`)
+    .then(response => response.json())
+}
```

Comment:
Nice refactor to make this more dynamic. I hadn't thought past iteration 2 for grabbing different kinds of data but this is a great set up for that!